library/liquibase

```diff
@@ -2,12 +2,12 @@ Maintainers: Jake Newton <docker@liquibase.com> (@jnewton03)
 Architectures: arm64v8, amd64
 GitRepo: https://github.com/liquibase/docker.git
 
-Tags: 4.28, 4.28.0, 4.28.0.1, latest
+Tags: , , latest
```

Comment:
Looks like this is an automation failure somewhere in the chain to making the PR and seems to be reflected in the PR title "Update library/liquibase to".

Reply:
cc maintainer: @jnewton03

Reply:
(see also https://github.com/docker-library/faq#can-i-use-a-bot-to-make-my-image-update-prs, especially point 2. which we should update to note CI failures :joy:)